src/components/StatusBar/StatusBar.tsx

```diff
@@ -48,6 +50,9 @@ export const StatusBar: FC<StatusBarProps> = ({
             Next
           </button>
           : null}
+        {!showNextButton && (
+          <button role="button" className="h5p-vocabulary-drill-restart" onClick={onRestart}>Restart</button>
```

Comment:
Why does the `button` element have a redundant aria role of `button`?

Reply:
Made a quick draft and managed to write `role` instead of `type`. Thanks for the correction

Reply:
Sorry, guess I was not clear. I wondered why there's an extra attribute at all. It can be useful if one really must use a `div` or something else than a `button`, but using a `button` should let browsers take care of the role/type situation already.

Reply:
It's recommended to always specify the type attribute of a `button` to tell browsers what type of `button` it is. That's why we add the type attribute.

Reply:
I was under the impression that setting `type` was only required inside a `form` where it would default to the value `submit`, but it seems I learned something today. Scrap my comment then.

Reply:
Seems that the `type` is only relevant if it's used in a `form` (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attributes) where it would then default so `submit`, but whatever ...